lib/src/model.dart

```diff
@@ -622,7 +622,7 @@ class PkgDependency implements Comparable<PkgDependency> {
 
   bool get isLatest => available == null;
 
-  bool get isOutdated => !isLatest;
+  bool get isOutdated => available != null && !available.isPreRelease;
```

Comment:
Can it happen that there is both a newer prerelease and a  newer nonprerelease available, and the prerelease shadows the non-prerelease making the dependency seem not outdated even though it is?

Reply:
The information is coming from the `pub` client, and we'll take here whatever it provides.

Any memory what `pub` uses? (latest stable, latest by date, or just highest version value?) I don't see much risk of using this at face value right now, but later on we could also query it from  from pub.dev.

Reply:
True, at worst we will warn too little.

I don't know off hand what the pub client does.
Perhaps add a TODO that we might want to investigate edge cases here.